src/components/Home/Home.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styles from "./Home.module.css";
+import RecentDecks from "../RecentDecks/RecentDecks";
+
+const Home = () => {
+  return (
+    <>
```

Comment:
Please wrap this piece of code with <Dashboard></Dashboard>, then in your routes config, change the component of route "/" to point to Home